llvm/lib/Target/AArch64/AArch64ISelLowering.h

```diff
@@ -1003,6 +1003,9 @@ class AArch64TargetLowering : public TargetLowering {
   /// make the right decision when generating code for different targets.
   const AArch64Subtarget *Subtarget;
 
+  llvm::BumpPtrAllocator bAlloc;
```

Comment:
LLVM style says member variables start with an uppercase letter

Reply:
Ah, sorry, I copied this from elsewhere